Classes/Systems/ShortcutHandler.swift

```diff
@@ -46,10 +51,17 @@ struct ShortcutHandler {
         let searchItem = UIApplicationShortcutItem(type: Items.search.rawValue,
                                                    localizedTitle: Constants.Strings.search,
                                                    localizedSubtitle: nil,
-                                                   icon: searchIcon,
-                                                   userInfo: nil)
+                                                   icon: searchIcon)
         items.append(searchItem)
 
+        // Bookmarks
+        let bookmarkIcon = UIApplicationShortcutIcon(templateImageName: "bookmarks")
+        let bookmarkItem = UIApplicationShortcutItem(type: Items.bookmarks.rawValue,
```

Comment:
nit: will fix this up myself, but should put all params on newline [like this](https://github.com/rnystrom/GitHawk/blob/master/Classes/Issues/IssuesViewController.swift#L118-L125)